action.yml

```diff
@@ -150,53 +159,50 @@ runs:
           fi
         }
 
-        # download and install yq
-        sudo wget -O /usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64
-        sudo chmod +x /usr/local/bin/yq
```

Comment:
yq is already part of the github action with ubuntu 20: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#tools